tests/suite/zar/s3/import.yaml

```diff
@@ -0,0 +1,27 @@
+script: |
+  export AWS_REGION=us-east-2 AWS_ACCESS_KEY_ID=minioadmin \
+        AWS_SECRET_ACCESS_KEY=minioadmin AWS_S3_ENDPOINT=http://localhost:9000
+  mkdir -p data/bucket
+  minio server --quiet --address "localhost:9000" ./data &> /dev/null &
+  pid=$!
+  until nc -vzw 2 localhost 9000; do sleep 2; done
```

Comment:
If minio fails to start, this test is going to fail without a clear indication of the problem. To improve on that, I think you'll need to stop redirecting the server's output to /dev/null and replace the until command with one that won't loop forever.

Reply:
Sure

Reply:
I'm not quite clear on what the "stop redirecting the server's output to /dev/null" will help with.

Reply:
Whoops, nevermind

Reply:
@nwt re-did this... PTAL!